src/cscout.cpp

```diff
@@ -3434,6 +3436,16 @@ main(int argc, char *argv[])
 	}
 #endif
 
+	if (process_mode == pm_r_option) {
+		cerr << "Producing call graphs" << endl;
+
+		for (const char *d : call_graphs) {
+			cerr << d << endl;
```

Comment:
Shouldn't these go to the specified files? Otherwise, shouldn't it be `cout`.

Reply:
This is for logging purposes so according to the philosophy it ought to go to stderr.

Reply:
OK, ignore.  I though it was the actual graph.